geode-core/src/distributedTest/java/org/apache/geode/cache/client/ClientServerRegisterInterestsDUnitTest.java

```diff
@@ -83,74 +79,62 @@ public final void postSetUp() throws Exception {
   public final void preTearDown() throws Exception {
     serverPort.set(0);
     entryEvents.clear();
-    gemfireServerVm.invoke(new SerializableRunnable() {
-      @Override
-      public void run() {
-        CacheFactory.getAnyInstance().close();
-      }
-    });
+    gemfireServerVm.invoke(() -> CacheFactory.getAnyInstance().close());
     gemfireServerVm = null;
   }
 
   private void setupGemFireCacheServer() {
-    Host localhost = Host.getHost(0);
 
-    gemfireServerVm = localhost.getVM(0);
+
+    gemfireServerVm = VM.getVM(0);
     serverPort.set(AvailablePortHelper.getRandomAvailableTCPPort());
 
-    gemfireServerVm.invoke(new SerializableRunnable() {
-      @Override
-      public void run() {
-        try {
-          Cache cache = new CacheFactory()
-              .set("name", "ClientServerRegisterInterestsTestGemFireServer").set(MCAST_PORT, "0")
-              .set(LOG_FILE, "clientServerRegisterInterestsTest.log").set(LOG_LEVEL, "config")
-              // .set("jmx-manager", "true")
-              // .set("jmx-manager-http-port", "0")
-              // .set("jmx-manager-port", "1199")
-              // .set("jmx-manager-start", "true")
-              .create();
+    gemfireServerVm.invoke(() -> {
+      try {
+        Cache cache = new CacheFactory()
+            .set("name", "ClientServerRegisterInterestsTestGemFireServer").set(MCAST_PORT, "0")
+            .set(LOG_FILE, "clientServerRegisterInterestsTest.log").set(LOG_LEVEL, "config")
+            .create();
 
-          RegionFactory<String, String> regionFactory = cache.createRegionFactory();
+        RegionFactory<String, String> regionFactory = cache.createRegionFactory();
 
-          regionFactory.setDataPolicy(DataPolicy.REPLICATE);
-          regionFactory.setKeyConstraint(String.class);
-          regionFactory.setValueConstraint(String.class);
+        regionFactory.setDataPolicy(DataPolicy.REPLICATE);
+        regionFactory.setKeyConstraint(String.class);
+        regionFactory.setValueConstraint(String.class);
 
-          Region<String, String> example = regionFactory.create("Example");
+        Region<String, String> example = regionFactory.create("Example");
 
-          assertNotNull("The 'Example' Region was not properly configured and initialized!",
-              example);
-          assertEquals(SEPARATOR + "Example", example.getFullPath());
-          assertEquals("Example", example.getName());
-          assertTrue(example.isEmpty());
+        assertNotNull("The 'Example' Region was not properly configured and initialized!",
+            example);
+        assertThat(example.getFullPath()).isEqualTo(SEPARATOR + "Example");
+        assertThat(example.getName()).isEqualTo("Example");
+        assertTrue(example.isEmpty());
 
-          example.put("1", "ONE");
+        example.put("1", "ONE");
 
-          assertFalse(example.isEmpty());
-          assertEquals(1, example.size());
+        assertFalse(example.isEmpty());
```

Comment:
For example `assertThat(example).isEmpty();`.

Reply:
fixed.